.github/workflows/test.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
         if: ${{ !inputs.coverage }}
         run: |
           source $VENV
-          pytest
+          pytest -m='not sqlalchemy_asyncpg'
```

Comment:
Should be the opposite, right? These are already disabled in `adopts` in `pyproject.toml`, so we'd want to explicitly run those here as well I think.